extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -559,15 +561,21 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 			blobTableText,
 			allFieldsRequiredLabel,
 			this._blobContainerTargetDatabaseNamesTable
-		]).component();
+		]).withProps({
+			CSSStyles: {
+				'display': 'none',
+			}
+		}).component();
 
 		const container = this._view.modelBuilder.flexContainer().withLayout({
 			flexFlow: 'column'
 		}).withItems([
 			this._networkTableContainer,
 			this._blobTableContainer
 		]).withProps({
-			display: 'none'
+			CSSStyles: {
+				'display': 'none',
+			}
```

Comment:
can this be converted to a style const and reused:  like hide_control_css, etc.

Reply:
this should be done in a separate pr as there will be a lot of changes outside of this pr context